cypress/integration/history.js

```diff
@@ -0,0 +1,39 @@
+describe('History Tests', () => {
+  beforeEach(() => {
+    // give a logged in editor and the site root
+    cy.autologin();
+    cy.visit('/');
+    cy.waitForResourceToLoad('@navigation');
+    cy.waitForResourceToLoad('@breadcrumbs');
+    cy.waitForResourceToLoad('@actions');
+    cy.waitForResourceToLoad('@types');
+    cy.waitForResourceToLoad('');
```

Comment:
I wonder if this does anything (I think I've introduced this style when I got rid of ?fullobjects).

Reply:
I hope it does!